packages/ui/src/styles.css

```diff
@@ -217,3 +217,13 @@
 .profile-card .description {
   @apply w-full text-xl font-normal leading-normal;
 }
+
+.btf {
+  @apply rounded-3xl border-none bg-[#44AF96] uppercase;
+  font-family: var(--font-future);
+}
+
+.bmtf {
```

Comment:
@okhaimie-dev these two css classes names means something? :thinking: 
`btf` stands for something? 

If we can consider defining more representatives naming for everything it is alway better thinking long term :wink: